Resources/doc/7-manual-route-definition.rst

```diff
@@ -1,128 +0,0 @@
-Manual definition of routes
-===========================
-
-If the automatic route generation does not fit your needs, you can manually
-define a route using simple annotations. This is very helpful if you want to
-have more than one url parameter without having a static word in between them.
-
-For a full list of annotations check out ``FOS/RestBundle/Controller/Annotations``:
-
-.. code-block:: php
-
-    // Delete Route Definition
-    use FOS\RestBundle\Controller\Annotations\Delete;
-
-    /**
-     * DELETE Route annotation.
-     * @Delete("/likes/{type}/{typeId}")
```

Comment:
all these examples are not that useful as the shortcuts are explained in the annotation-reference doc already